src/App.js

```diff
@@ -1,25 +1,63 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  NavLink,
+} from 'react-router-dom';
+import AddItem from './components/AddItem';
+import List from './components/List';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div
+        className="App"
+        style={{
+          display: 'flex',
+          flexFlow: 'column nowrap',
+          justifyContent: 'flex-start',
+          alignItems: 'center',
+          minHeight: '100vh',
+          position: 'relative',
+        }}
```

Comment:
I'm curious about this layout ^ 👀  . What led you to use this CSS, in particular?

Reply:
We initially tried to use flexbox to push navigation to the bottom, but it didn't look good for the items in the middle, hence the complications. I can see now that these styles don't make much sense together and we can achieve the same result by just styling the `nav` with `bottom: 0px; left: 50%;`

Reply:
I _think_ you can remove these styles, give the `nav` `width: 100%`, then display flex the `ul`, and `justify-content: space-between` or `space-evenly`...let me verify.

Reply:
yes. This is with getting rid of those inline styles and with `space-evenly` (you could also do `padding: 0` on the `ul`. It has a default padding-left of 15px. :
![image](https://user-images.githubusercontent.com/10378331/113797852-adbfc400-9717-11eb-8d68-11f8a5e411bd.png)

Reply:
We totally were overthinking that 😂. Thanks for pointing that out Lauren!